interface/patient_file/summary/pnotes_fragment.php

```diff
@@ -76,38 +84,30 @@
             echo "<td valign='top' class='text' ><b>". htmlspecialchars(xl('Date'), ENT_NOQUOTES) ."</b></td>\n";
             echo "<td valign='top' class='text' ><b>". htmlspecialchars(xl('Subject'), ENT_NOQUOTES) ."</b></td>\n";
             echo "<td valign='top' class='text' ><b>". htmlspecialchars(xl('Content'), ENT_NOQUOTES) ."</b></td>\n";
-            echo "<td valign='top' class='text' ><b>". htmlspecialchars(xl('Status'), ENT_NOQUOTES) ."</b></td>\n";
+            echo "<td valign='top' class='text' ></td>\n";
             echo "</tr>\n";
             foreach ($result as $iter) {
                 $has_note = 1;
 
                 $body = $iter['body'];
-                if (preg_match('/^\d\d\d\d-\d\d-\d\d \d\d\:\d\d /', $body)) {
-                    $body = nl2br(htmlspecialchars(oeFormatPatientNote($body), ENT_NOQUOTES));
-                } else {
-                    $body = htmlspecialchars(oeFormatSDFT(strtotime($iter['date'])) . date(' H:i', strtotime($iter['date'])) .
-                    ' (' . $iter['user'] . ') ', ENT_NOQUOTES) .
-                    nl2br(htmlspecialchars(oeFormatPatientNote($body), ENT_NOQUOTES));
-                }
-
+                $body = preg_replace('/(\d{4}-\d{2}-\d{2} \d{2}:\d{2}\s\([^)(]+\s)(to)(\s[^)(]+\))/', '', $body);
                 $body = preg_replace('/(\sto\s)-patient-(\))/', '${1}'.$patientname.'${2}', $body);
-                $body = strlen($body) > 120 ? substr($body, 0, 120)."<b>.......</b>" : $body;
                 echo " <tr class='text' id='".htmlspecialchars($iter['id'], ENT_QUOTES)."' style='border-bottom:1px dashed;height:30px;' >\n";
 
                 // Modified 6/2009 by BM to incorporate the patient notes into the list_options listings
                 echo "<td valign='top' class='text'>".htmlspecialchars($iter['user'], ENT_NOQUOTES)."</td>\n";
-                echo "<td valign='top' class='text'>".htmlspecialchars($iter['date'], ENT_NOQUOTES)."</td>\n";
+                echo "<td valign='top' class='text'>".htmlspecialchars($iter['assigned_to'], ENT_NOQUOTES)."</td>\n";
+                echo "<td valign='top' class='text'>".htmlspecialchars(date('Y-m-d H:i', strtotime($iter['date'])), ENT_NOQUOTES)."</td>\n";
                 echo "  <td valign='top' class='text'><b>";
                 echo generate_display_field(array('data_type'=>'1','list_id'=>'note_type'), $iter['title']);
                 echo "</b></td>\n";
 
                 echo "  <td valign='top' class='text'>$body</td>\n";
-                echo "  <td valign='top' class='text'>";
-                echo generate_display_field(array('data_type'=>'1','list_id'=>'message_status'), $iter['message_status']);
-                echo "</td>\n";
+                echo "<td valign='top' class='text'><button data-id='" . $iter['id'] . "' class='complete_btn'>" . htmlspecialchars('Completed', ENT_NOQUOTES) . "</button></td>\n";
```

Comment:
translate Completed:
``````
xlt('Completed')

Reply:
Also should escape $iter['id']
``````
attr($iter['id'])

Reply:
ok done, don't know how I missed that...